README.md

```diff
@@ -1 +1,220 @@
+```sh
+  git clone https://github.com/lily4178993/react-todo-app.git
+```
+  - Navigate to the folder
+```
+cd react-todo-app
+code .
+```
+- Checkout the branch
+```
+git checkout dev
+```
+<br><hr>
+
+<!-- INSTALL -->
+### <b>Install 🏗️</b><a name="install"></a>
+
+Install all dependencies:
+
+```sh
+    npm install
+```
+<br><hr>
+
+<!-- USAGE -->
+### <b>Usage 📂</b><a name="usage"></a>
+
+To run the project, open your terminal and tap the following line.
+```sh
+    npm start
+```
+
+<!-- TESTS -->
+<br><hr>
+### <b>Run tests 🧪</b><a name="run-tests"></a>
+
+How should it be tested? 
+To run tests, run the following command:
+
+- CSS linter errors run:
+
+```sh
+npx stylelint "**/*.{css,scss}" --fix
+```
+
+- JavaScript linter errors run:
+
+```sh
+npx eslint "**/*.{js,jsx}" --fix
+```
+
+<br><hr>
+
+<!-- DEPLOYEMENT -->
+### <b>Deployment</b> <a name="deployment"></a>
+
+> This app will be deployed in the near future.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br><hr>
+
+<!-- AUTHORS -->
+## 👤 Author <a name="author"></a>
+
+**Nelly Telli**
+- GitHub: [@lily4178993](https://github.com/lily4178993/)
+- LinkedIn: [Nelly T.](https://www.linkedin.com/in/nelly-t-330414266/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br><hr>
+
+<!-- FUTURE FEATURES -->
+## <b>Future Features 🔭</b><a name="future-features"></a>
+
+- **Add About and Contact Pages**
+- **Add editing functionalities to Todos list items**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br><hr>
+
+
+<!-- CONTRIBUTING -->
+
+## <b>Contributing 🤝</b><a name="contributing"></a>
+
+If you have suggestions 📝, ideas 🤔, or bug reports 🐛, please feel free to open an [issue](https://github.com/lily4178993/react-todo-app/issues) on GitHub.
+Remember, every contribution, no matter how big or small, makes a difference.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br><hr>
+
+
+
+<!-- SUPPORT -->
+## <b>Show your support 🌟</b><a name="support"></a>
+
+Thank you for taking the time to explore my GitHub project! Your support means a lot to me. If you find my project valuable and would like to contribute, here are a few ways you can support me:
+
+ - Star the project ⭐️
+ - Fork the project 🍴 🎣
+ - Share with others 🗺️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br><hr>
+
+<!-- ACKNOWLEDGEMENTS -->
+## <b>Acknowledgments 🙏</b><a name="acknowledgements"></a>
+
+I would like to express my sincere gratitude to [Microverse](https://github.com/microverseinc), the dedicated reviewers, and my [partner](https://github.com/SanaTameem).
+
+I would also like to thank <a target="_blank" href="https://icons8.com">Icons8</a> for all the images and icons I used in this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br><hr>
+
+<!-- LICENCE -->
+## <b>License 📝</b><a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- Kindly try to add a correct link for your MIT file.

Reply:
Hi @SanaTameem. The link for the License file is correct.